lib/rules/selector-max-type/README.md

```diff
@@ -115,6 +115,19 @@ div span a.foo {}
 div span a#bar {}
 ```
 
+#### `"custom-elements"`
+
+Discount custom elements.
+
+For example, with `2`:
+
+The following patterns is _not_ considered a problem:
```

Comment:
[trivial] Can you please fix the typo?

```suggestion
The following pattern is _not_ considered a problem:
```

Reply:
Fixed now. Thanks for noticing, sorry  I did not catch it earlier